tools/rdiff-backup.spec.template-fedora

```diff
@@ -42,75 +37,31 @@ securely back a hard drive up to a remote location, and only the
 differences from the previous backup will be transmitted.
 
 %prep
-%setup -q
+%autosetup -n %{name}-%{version} -p1
 
 %build
-%{PYTHON_NAME} setup.py build
+%py3_build
 
 %install
-%{PYTHON_NAME} setup.py install --root $RPM_BUILD_ROOT
-# Produce .pyo files for %ghost directive later
-%{PYTHON_NAME} -Oc 'from compileall import *; compile_dir("'$RPM_BUILD_ROOT/%{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup'")'
-
-%clean
-rm -rf $RPM_BUILD_ROOT
+%py3_install
 
 %files
 %defattr(-,root,root)
 %{_bindir}/rdiff-backup
 %{_bindir}/rdiff-backup-statistics
 %{_bindir}/rdiff-backup-delete
 %{_mandir}/man1/rdiff-backup*
-%dir %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup
-%{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/*.py
-%{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/__pycache__/*.pyc
-%{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/*.so
-%{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup-*-py?.?.egg-info
+%{python3_sitearch}/rdiff_backup/
+%{python3_sitearch}/rdiff_backup-*.egg-info
+%{python3_sitearch}/rdiffbackup/
 %{_datadir}/bash-completion/completions/rdiff-backup
-%ghost %{_libdir}/python%{PYTHON_VERSION}/site-packages/rdiff_backup/__pycache__/*.pyo
-%doc CHANGELOG.adoc COPYING README.adoc docs/*.adoc
+%doc CHANGELOG.adoc README.adoc
+%doc docs/credits.adoc docs/DEVELOP.adoc docs/examples.adoc
+%doc docs/FAQ.adoc docs/migration.adoc
+%doc docs/Windows-README.adoc docs/Windows-DEVELOP.adoc
```

Comment:
I don't think we need to put the Windows documentation in the Linux package

Reply:
True, although it is actually me being lazy, as otherwise I need to remove them in the install statement.
I'll take them out, as I noted in the Fedora packaging guidelines, it also says to do not include "non-essential" docs, such as those for Windows.